fastlane/lanes/release.rb

```diff
@@ -457,14 +506,17 @@
 # @param [Boolean] beta Indicate if we should build a beta or regular release
 #
 def trigger_buildkite_release_build(branch:, beta:)
-  buildkite_trigger_build(
+  build_url = buildkite_trigger_build(
     buildkite_organization: BUILDKITE_ORGANIZATION,
     buildkite_pipeline: BUILDKITE_PIPELINE,
     branch: branch,
     environment: { BETA_RELEASE: beta },
     pipeline_file: 'release-builds.yml',
     message: beta ? 'Beta Builds' : 'Release Builds'
   )
+
+  message = "This build triggered a build on <code>#{branch}</code>:<br>- #{build_url}"
```

Comment:
Nitpick: Other messages us Markdown formatting.

```suggestion
  message = "This build triggered a build on `#{branch}`:\n\n- #{build_url}"
```

Reply:
I've updated the comment https://github.com/wordpress-mobile/WordPress-iOS/pull/23585#discussion_r1766222155 to bundle this change together with the early return.